CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to contribute
+
+What to do? There is a companion project where the feature backlog lives. Go to the issues in [https://github.com/softwerkskammer/Agora-Backlog/issues](https://github.com/softwerkskammer/Agora-Backlog/issues)
+
```

Comment:
This is by the way also very very outdated. I think we should remove this information for now. (and maybe groom or delete the backlog project...)